parlai/crowdsourcing/utils/tests.py

```diff
@@ -8,30 +8,49 @@
 """
 
 import os
+import random
 import tempfile
 import time
-import unittest
 from typing import Any, Dict, List, Optional
 
+import numpy as np
+import torch
 from hydra.experimental import compose, initialize
 from mephisto.abstractions.databases.local_database import LocalMephistoDB
 from mephisto.operations.operator import Operator
 from mephisto.abstractions.blueprint import SharedTaskState
 from mephisto.tools.scripts import augment_config_from_db
+from pytest_regressions.data_regression import DataRegressionFixture
 
 
-class AbstractCrowdsourcingTest(unittest.TestCase):
+class AbstractCrowdsourcingTest:
```

Comment:
Are you removing the `unittest.TestCase` portion to ensure that this isn't seen as a test? It's possible to instead add a `setUpClass()` method to skip if this is the base class, like:
```
    @classmethod
    def setUpClass(cls):
        if cls is AbstractCrowdsourcingTest:
            raise unittest.SkipTest("Skip AbstractCrowdsourcingTest tests, it's a base class")
        super(AbstractCrowdsourcingTest, cls).setUpClass()
```
If not, I'm interested to hear your use case, as maybe I should be doing differently in Mephisto. This also may not be the best approach, more starting a discussion.

I'm betting it's more likely to do with `pytest.fixtures`, which I'd be happy to hear about your experience with afterwards.

Reply:
Yeah, I'm doing this because I'm gradually shifting all of the tests over to `pytest.fixtures`, so I can get more flexibility over whether the setup code runs once per class or once per function. I think it's a powerful tool but it requires a bit of fiddling to get right - the other good thing about it is that it'll enable me to use pytest regressions to avoid having to manually specify the desired outputs of test code.